### ISSUE TITLE
QUESTION: Does vulcand support SSL client authentication towards a backend server?

### ISSUE THREAD

**Issue by MichaelStephan** [mailgun/vulcand#186]
Hello,

does vulcand support SSL client authentication towards a backend server? 

 Thanks for your answer.

Regards,
Michael Stephan

**Comment by porjo** [comment]
No, I don't believe so. Out of curiosity, why is this required?

**Comment by grantr** [comment]
This might be what you're looking for: http://docs.vulcand.io/proxy.html#https-backends
